### ISSUE TITLE
Mismatch between pod spec and Cocoapods Specs repository

### ISSUE THREAD

**Issue by larslockefeer** [jflinter/Dwifft#41]
When trying to install Dwifft through Cocoapods in an app that has iOS 8 as the minimum deployment target, I get the following error:

```
[!] Unable to satisfy the following requirements:

- `Dwifft (~> 0.4)` required by `Podfile`
- `Dwifft (~> 0.4)` required by `Podfile`

Specs satisfying the `Dwifft (~> 0.4)` dependency were found, but they required a higher minimum deployment target.
```

Initially I thought Dwifft's deployment was higher than iOS 8, but when I checked the [Podspec here in this repository](https://github.com/jflinter/Dwifft/blob/master/Dwifft.podspec), I saw that it has iOS 8 as the minimum deployment version. Nevertheless problems persisted.

However, when I add Dwifft in my Podfile as follows:

```
 pod 'Dwifft', :git => 'git@github.com:jflinter/Dwifft.git'
```

it just works™.

This got me thinking that maybe there is a mismatch between the current Podspec in this repository and the Podspec that is published to the CocoaPods specs repo. Looking at [spec for Dwifft in the CocoaPods Specs repo](https://github.com/CocoaPods/Specs/blob/master/Specs/Dwifft/0.4/Dwifft.podspec.json), this seems to indeed be the case:

```
...
"platforms": {
    "ios": "9.0"
  },
...
```

Could it be that the minimum deployment target was changed after pushing the spec to the CocoaPods Specs repo? In that case, bumping the version number and pushing again would suffice.

**Comment by larslockefeer** [comment]
I only now notice that this is a duplicate of #32, sorry for that.

**Comment by jflinter** [comment]
Thanks for opening @larslockefeer - I've resolved this as of version 0.5.
